pkgs/development/libraries/ogdf/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, cmake, doxygen }:
+
+stdenv.mkDerivation rec {
+  name = "ogdf-${version}";
+  version = "2020.02";
+  src = fetchFromGitHub {
+    owner = "ogdf";
+    repo = "ogdf";
+    rev = "be4452b6c36575d11a04d5bb1b362edeb8173eaa";
+    sha256 = "0drrs8zh1097i5c60z9g658vs9k1iinkav8crlwk722ihfm1vxqd";
+  };
+
+  hardeningDisable = [ "all" ];
+  nativeBuildInputs = [ cmake doxygen ];
+  cmakeFlags = [ "-DOGDF_WARNING_ERRORS=OFF"
```

Comment:
What is the reason to use this flag?

Reply:
thanks for catching this. no reason and i removed it.